austation/code/modules/hydroponics/grown.dm

```diff
@@ -1,3 +1,19 @@
+/obj/item/reagent_containers/food/snacks/grown
+	var/mutable_appearance/grown_overlay //Used for 'Floral Spines' plant gene
+	var/mob/living/target
```

Comment:
these vars could all be moved to the plant gene itself, rather than being on the plants

Reply:
It's easier in terms of initializing.